android/app/src/main/java/com/example/weatherapp/AppViewModel.kt

```diff
@@ -86,22 +86,22 @@ class AppViewModel @Inject constructor(
         }
     }
 
-    @SuppressLint("SimpleDateFormat")
-    fun sprintDateFormat(day: ForecastDay): String {
-        val calendar = Calendar.getInstance()
-        calendar.add(Calendar.DATE, day.day)
-        return when (day) {
-            ForecastDay.TODAY -> {
-                val formatter = SimpleDateFormat("yyyy-MM-dd HH:mm")
-                formatter.format(calendar.time)
-            }
+@SuppressLint("SimpleDateFormat")
+fun sprintDateFormat(day: ForecastDay): String {
+    val calendar = Calendar.getInstance()
+    calendar.add(Calendar.DATE, day.day)
+    return when (day) {
+        ForecastDay.TODAY -> {
+            val formatter = SimpleDateFormat("yyyy-MM-dd HH:mm")
+            formatter.format(calendar.time)
+        }
```

Comment:
🚫 **[ktlint]** <standard:indent> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unexpected indentation (8) (should be 12)